Bottom: f0e4866a814196281dfc54c14c41aaa89ae2474a
Top:    24a56ae533292305bd5cbfc8ec538375b8872565
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 09:44:03 +0800

Refresh of autofs-5.0.7-fix-typo-forced-shutdown-should-be-force-shutdown.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 3228d6b..5751805 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -57,6 +57,7 @@
 - fix a couple of compiler warnings.
 - add after sssd dependency to unit file.
 - dont start readmap unless ready.
+- fix typo forced-shutdown should be force-shutdown.
 
 25/07/2012 autofs-5.0.7
 =======================
